client/src/navigation/RouterConfig.tsx

```diff
@@ -21,17 +24,25 @@ const EventAdmin = React.lazy(() => import('./EventAdminContainer'));
 //                        url     label   component
 export type RouteTuple = [string, string, React.ComponentType<{}>];
 export type RouteList = Array<RouteTuple>;
+export const registerUrl = '/events/:eventId/register';
 
 const RouterConfig = () => {
   const { pathname } = useLocation();
 
   return (
     <Switch>
       <Redirect from="/:url*(/+)" to={pathname.slice(0, -1)} />
-      <Route exact path="/" component={Default} />
-      <Route exact path="/events/:eventId/register" component={RegisterPage} />
+      <Provider store={regStore}>
+        <Route exact path="/" component={Default} />
+        <Route exact path={registerUrl}>
+          <Redirect to={`${pathname}/form${window.location.search}`} />
```

Comment:
This redirect allows the existing links to `/events/:eventId/register` to continue to work, and even passes the search params along, so that emailed links to special registrations to continue working

Reply:
Maybe this should match the the old jargon in the backend, rather than inventing new page names?